README.md

```diff
@@ -218,15 +218,15 @@ and the new state of `downshift`. (see `onStateChange` for more info on
 `stateAndHelpers`).
 
 - `selectedItem`: The item that was just selected
-- `stateAndHelpers`: This is the same thing your `children` prop
+- `stateAndHelpers`: This is the exact same thing you're `children` prop
```

Comment:
This should be "your". Also, this line is inconsistent with line 238. We should either keep "exact" or take it out, but it should be consistent.